syft/frameworks/torch/tensors/interpreters/large_precision_tensor.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import torch
+
+from syft.frameworks.torch.overload_torch import overloaded
+from syft.frameworks.torch.tensors.interpreters import AbstractTensor
+
+
+class LargePrecisionTensor(AbstractTensor):
+    """LargePrecisionTensor allows handling of numbers bigger than LongTensor
+
+    Some systems using Syft require larger types than those supported natively. This tensor type supports arbitrarily
+    large values by packing them in smaller ones.
+    If the original tensor is a PyTorch tensor it is not processed
+    """
+
+    def __init__(
+        self, owner=None, id=None, tags=None, description=None, precision=16, virtual_prec=0
```

Comment:
why were these chosen as defaults

Reply:
also add typing

Reply:
I used `16` bits as the precision that would be needed when moving this tensor to other systems (I had JVM in mind).
It would be good to have this value set in a global configuration of the setup.
I am also thinking that instead of using a number we could use a `dtype`. I will give it a try and see how it looks like

Reply:
Using `dtype` now